ydb/docs/en/core/yql/reference/syntax/update.md

```diff
@@ -49,3 +49,19 @@ $to_update = (
 UPDATE my_table ON
 SELECT * FROM $to_update;
 ```
+
+## BATCH UPDATE {#batch-update}
+
+Updates large tables that the standard `UPDATE` cannot update. It independently applies changes to each partition of the specified table, processing a limited number of rows per iteration (by default, 10000 rows). The query is executed in a non-transactional mode. In case of an error, changes are not rolled back. The semantics are inherited from the standard `UPDATE` with the following restrictions:
```

Comment:
cannot update .... because of the transactional limits and the risk of potential transaction locks invalidation failure

?